docs/zh-cn/ops/deploy-by-docker-compose.md

```diff
@@ -453,7 +453,7 @@ services:
     image: seataio/seata-server:${latest-release-version}
     ports:
       - "7092:7091"
-      - "8092:8091"
+      - "8092:8092"
```

Comment:
yes, you are right. This container exposes port 8092 through environment variables `SEATA_PORT=8092`